src/lib/features/playback-controls/ui/playback-controls.svelte

```diff
@@ -0,0 +1,34 @@
+<script>
+  import { paused, move } from '$lib/entities/audio';
+  import { toggleGlobalPlayback } from '..';
+  import { IconButton } from '$lib/shared/ui/';
+  import PlaybackButton from './playback-button.svelte';
+  import IconReplay10 from '~icons/ic/round-replay-10';
+  import IconForward30 from '~icons/ic/round-forward-30';
+
+  export let prominent = false;
```

Comment:
question: is this component used anywhere without this prop? I think we reverted the mini-player, so this prop isn't really necessary anymore